src/inc/zapper.h

```diff
@@ -126,7 +126,11 @@ class Zapper
 
 #if defined(FEATURE_CORECLR) && !defined(FEATURE_MERGE_JIT_AND_ENGINE)
     SString                 m_CLRJITPath;
+    bool                    m_fDontLoadJit;
```

Comment:
I'll add a check to disallow specifying JITPath when CreatePDB. This bool variable is still needed to control whether or not to load JIT (basically, don't load JIT if creating PDB, load JIT otherwise, independent of whether JITPath is specified).